saleor/webhook/transport/synchronous/circuit_breaker/breaker_board.py

```diff
@@ -41,12 +89,21 @@ def register_error(self, app_id: int):
             f"{0}-{1}".format(app_id, "error"), self.ttl
         )
 
-        if errors >= self.failure_threshold:
+        open = False
+        if self.mode == BreakerMode.FIXED and errors >= self.failure_threshold:
+            open = True
+        elif self.mode == BreakerMode.PERCENTAGE:
+            total = self.storage.get_event_count("success") + errors
+            if errors * 100 / total >= self.failure_threshold:
```

Comment:
For `PERCENTAGE` strategy we need to take into account:
- ✅ percentage (already implemented here)
- minimum number of errors that need to happen for circuit breaker to open, for example:
  - 10 webhooks happened, 8 failed - 80% of webhooks failed - this might not be enough of justification to open breaker
  - 1000 happened, 500 failed - 50% of webhooks failed - this might be enough of justification to open breaker